tests/network-ovn

```diff
@@ -716,10 +716,10 @@ ovn_forward_tests() {
     lxc network forward create ovn-virtual-network 2001:db8:1:2::1 target_address="${U1_IPV6}"
     sleep 2
 
-    # Check forward is working by pinging it.
+    # Check forward is working by probing TCP/22.
```

Comment:
We should add the ping test as a separate (commented out for now) test with a comment explaining its an ovn regression, then hopefully we can re-enable later.